eradiate/scenes/measure.py

```diff
@@ -388,24 +389,21 @@ class PerspectiveCameraMeasure(Measure):
 
         Unit-enabled field (default: cdu[length]).
 
-    ``zenith`` (float):
-        Zenith angle. Default value: 0 deg.
-
-        Unit-enabled field (default: cdu[angle]).
-
-    ``azimuth`` (float):
-        Azimuth angle value. Default value: 0 deg.
+    ``origin`` (array[float]):
+        A 3-element vector specifying the position of the camera.
+        Default: [0, 0, 0] m.
```

Comment:
Your defaults are currently `origin == target`: this will result in default params making `kernel_dict()` raise (BTW I think the origin-target distance, if checked, should be checked during the init sequence, not when expanding to dictionary). Maybe select a default value positioning `origin` a couple dozen metres aways from `target`?